🐛 [firebase_analytics] FlutterFirebaseAnalyticsPlugin has unchecked cast

## Bug report

**Describe the bug**
Build is throwing warnings. compiling with details shows it is an unchecked cast within FlutterFirebaseAnalyticsPlugin.java


```
/Users/Applications/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_analytics-9.1.7/android/src/main/java/io/flutter/plugins/firebase/analytics/FlutterFirebaseAnalyticsPlugin.java:69: warning: [unchecked] unchecked cast
            list.add(createBundleFromMap((Map<String, Object>) item));
                                                               ^
  required: Map<String,Object>
  found:    Object


/Users/Applications/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_analytics-9.1.7/android/src/main/java/io/flutter/plugins/firebase/analytics/FlutterFirebaseAnalyticsPlugin.java:82: warning: [unchecked] unchecked cast
        bundle.putParcelable(key, createBundleFromMap((Map<String, Object>) value));
                                                                            ^
  required: Map<String,Object>
  found:    Object

```
### Steps to reproduce

Steps to reproduce the behavior:

1. Compiled with android studio on mac

---

### Flutter doctor

Run `flutter doctor` and paste the output below:

<details><summary>Click To Expand</summary>

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.5, on macOS 12.2.1 21D62 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.1)
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.66.2)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

```

</details>

---

### Flutter dependencies

using:
  firebase_core: ^1.16.0
  firebase_analytics: ^9.1.7
  firebase_auth: ^3.3.17
  cloud_firestore: ^3.1.14

  firebase_crashlytics: ^2.7.2
  firebase_messaging: ^11.3.0
  firebase_performance: ^0.8.0+11

---

--- a/packages/firebase_analytics/firebase_analytics/android/src/main/java/io/flutter/plugins/firebase/analytics/FlutterFirebaseAnalyticsPlugin.java
+++ b/packages/firebase_analytics/firebase_analytics/android/src/main/java/io/flutter/plugins/firebase/analytics/FlutterFirebaseAnalyticsPlugin.java
@@ -40,6 +40,7 @@ public class FlutterFirebaseAnalyticsPlugin
     FlutterFirebasePluginRegistry.registerPlugin(channelName, this);
   }
 
+  @SuppressWarnings("unchecked")
   private static Bundle createBundleFromMap(Map<String, Object> map) {
     if (map == null) {
       return null;

